src/js/components/Calendar/__tests__/__snapshots__/Calendar-test.js.snap

```diff
@@ -17230,7 +17277,11 @@ exports[`Calendar select dates 1`] = `
         </div>
       </div>
       <div
-        aria-label="January 2020"
+        aria-label="
+                January 2020;
+                Currently selected 
+  2020-01-12 ,Range: 2020-01-08 ,2020-01-10 
```

Comment:
I'm not sure I understand the difference between "currently selected" vs "range" -- is this which date is active vs which dates have been selected previously for range?

Reply:
there isn't a difference between currently selected and range. in this test, as long as i'm understanding it correctly, january 12 is selected and the range from january 8th to january 10th is also selected. so something like this situation: 

<img width="500" alt="Screen Shot 2021-09-10 at 5 08 39 PM" src="https://user-images.githubusercontent.com/38971752/132917935-0c5ea96c-46c8-436d-ae74-47a9aa807611.png">

so maybe i could better phrase it so it's clear that all of the read aloud dates are selected? what do you think?

Reply:
Got it -- maybe it could read "2020-01-08 through 2020-01-10, 2020-01-12"? 

Unless other calendars have common convention of saying "Range:" to screen reader users, but my though was to keep it more conversational if possible.